subsystem_controllers/config/guidance_controller_config.yaml

```diff
@@ -59,3 +59,4 @@
     # List of guidance plugins that are ported to ROS2. If not in this list, it is assumed to be ROS1, and not managed.
     ros2_initial_plugins:
       - /guidance/plugins/inlanecruising_plugin
+      - /guidance/plugins/cooperative_lanechange
```

Comment:
Add this to the config as well and the required_plugins list

Reply:
Confirmed that the plugin is already listed in both SubsystemControllerParams.yaml within carma-config as well as the required_plugins_list.

Reply:
Created carma-config PR [#209](https://github.com/usdot-fhwa-stol/carma-config/pull/209) to address this.